src/app/multi-editor/multi-editor.component.ts

```diff
@@ -61,12 +60,11 @@ export class MultiEditorComponent implements OnInit {
   }
 
   onSave() {
-    this.queryService.save(this.previewedActions, this.checkedRecords)
-    .then((res) => {
-      this.successMessage = res['message'];
-      this.totalRecords = -1;
-      this.changeDetectorRef.markForCheck();
-    })
+    let checkedUUIDs: string[];
+    this.allSelected ? checkedUUIDs = [] :
+    checkedUUIDs = Object.keys(this.recordSelectionStatus)
+    .filter(key => this.recordSelectionStatus[key]);
+    this.queryService.save(this.previewedActions, checkedUUIDs, this.allSelected)
```

Comment:
now don't pass all selected.

Reply:
also careful with `get` with computation inside, only in this function ` get allSelected()` executed twice.